src/parkapi_sources/converters/base_converter/push/normalized_xlsx_converter.py

```diff
@@ -62,6 +72,23 @@ class NormalizedXlsxConverter(XlsxConverter, ABC):
         'Am Straßenrand': 'ON_STREET',
     }
 
+    def get_mapping_by_header(self, row: tuple[Cell]) -> dict[str, int]:
```

Comment:
Why did you introduce `row_values_no_special_chars`? Don't see any reason for this, as `{'some.count': 'capacity'}` is a perfectly valid dictionary which will work for the mapping.

Reply:
I would just redefined the ```header_row``` as a whole to avoid complications.